course/include/isometry.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <string>
+#include <iostream>
+#include <cmath>
+
+namespace cppcourse
+{
+
+class Vector3
+{
+public:
+  Vector3(const std::initializer_list<double> &rhs);
+  Vector3(const double &x = 0., const double &y = 0., const double &z = 0.) : x_(x),
+                                                                              y_(y),
+                                                                              z_(z){};
+  int value() const { return value_; }
+  Vector3 operator+(const Vector3 &other) const;
+  Vector3 operator-(const Vector3 &other) const;
+  Vector3 operator/(const Vector3 &other) const;
+  Vector3 operator*(const double &other) const;
+  Vector3 operator*(const Vector3 &other) const;
+  double dot(const Vector3 &other) const;
+
+  bool operator==(const double other[3]) const;
+  bool operator!=(const Vector3 &other) const { return (not(*this == other)); }
```

Comment:
First time ever that I see the keyword form of the operator `not` actually used. A few months back I did not even know it existed such a form.

Alternative operators are standard, so it's ok to use them, but be aware that it's not really frequent. It's like talking Australian english in the USA: you can get away with it, but it'll get noticed. Also be aware that in some companies using this may break away from the style guide.

Some companies may prefer that you write code in the same style as the

Reply:
Changed to operator `!`